apps/vr-tests-react-components/src/stories/Rating.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { Rating } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('Rating Converged Color', module)
```

Comment:
I think you can have multiple `storiesOf` with the same name.
```suggestion
storiesOf('Rating Converged', module)
```